index.js

```diff
@@ -1,20 +1,39 @@
 const express = require('express');
 const cors = require('cors');
 const sequelize = require('./config/connection');
-
 const app = express();
+const http = require('http');
+const server = http.createServer(app);
+const { Server } = require('socket.io');
+
+const io = new Server(server, {
+    cors: {
+        cors: 'http://localhost:3000',
```

Comment:
this will break CORS for anyone not calling it from localhost:3000 (the netlify app will break)